### PR TITLE
RNs - Minor Edits

### PR DESCRIPTION
**Pull request by janettelynch**
Capitalization consistency changes and active voice changes.

### CODE REVIEW

File: release-notes/2021-79.md [hunk 1]
```diff
@@ -94,29 +95,6 @@ September 10, 2021
 
 - Improved error messaging in the UI when bulk synchronization fails.
 
-## Dashboards
```
Comment (erkangz) [inline release-notes/2021-79.md:97]: why are these deleted?

Comment (janettelynch) [reply release-notes/2021-79.md:97]: @erkangz Sorry. I hadn't merged main to my branch or reviewed those additions from last night. But I found a spelling error and made a couple of edits. :)

Comment (erkangz) [reply release-notes/2021-79.md:97]: np. I think your latest commit overwrote a fix I put in

Comment (janettelynch) [reply release-notes/2021-79.md:97]: I should have had a conflict when I pushed, no? Can I put it back in for you or do you want to do it?

Comment (janettelynch) [reply release-notes/2021-79.md:97]: @erkangz I put the spaces back in for the admonition text.

File: release-notes/2021-79.md [hunk 2]
```diff
@@ -139,9 +134,9 @@ Stay tuned as additional dashboards, alert rules, managed questions, and complai
   `aws_kms_key` entity.
 
   !!! note 
-   This change requires the additional `ec2:GetEbsDefaultKmsKeyId` and
-  `ec2:GetEbsEncryptionByDefault` permissions in the
-  `JupiterOneSecurityAudit` IAM policy of each AWS account being monitored.
+      This change requires the additional `ec2:GetEbsDefaultKmsKeyId` and
+     `ec2:GetEbsEncryptionByDefault` permissions in the
+     `JupiterOneSecurityAudit` IAM policy of each AWS account being monitored.
```
Comment (erkangz) [inline release-notes/2021-79.md:139]:
```suggestion
      This change requires the additional `ec2:GetEbsDefaultKmsKeyId` and
      `ec2:GetEbsEncryptionByDefault` permissions in the
      `JupiterOneSecurityAudit` IAM policy of each AWS account being monitored.
```

**Review by erkangz — APPROVED** [review]
